cmd/car/testdata/script/get-block.txt

```diff
@@ -0,0 +1,19 @@
+env SAMPLE_CID='bafy2bzacebohz654namrgmwjjx4qmtwgxixsd7pn4tlanyrc3g3hwj75hlxrw'
+env MISSING_CID='bafy2bzacebohz654namrgmwjjx4qmtwgxixsd7pn4tlanyrc3g3hwj75xxxxx'
+
+# "get-block" on a CARv1 with an output file.
+car get-block ${INPUTS}/sample-v1.car ${SAMPLE_CID} out.block
```

Comment:
can we re-use the sample cars in the other parts of this same repo?

Reply:
Unfortunately we cannot, because each module is zipped separately. If we make them share testdata files across module boundaries, then the tests only pass if you've got a full `git clone`.

Reply:
I think this is fine in practice because the library needs lots of different input files to exercise error cases, but I think the CLI should only need 2-3 good input files to test happy paths.